html_table_parser/parser.py

```diff
@@ -52,7 +56,7 @@ def handle_endtag(self, tag):
             self._in_th = False
 
         if tag in ['td', 'th']:
-            final_cell = " ".join(self._current_cell).strip()
+            final_cell = "".join(self._current_cell).strip()
```

Comment:
Ok. I especially like that a separator is now configurable for the stripped tags. Thank you!